tools/tune/src/src_tiny_int16.m

```diff
@@ -27,14 +27,15 @@
 % Author: Seppo Ingalsuo <seppo.ingalsuo@linux.intel.com>
 %
 
-fs1 = [8e3 16e3 32e3 44.1e3 48e3 50e3];
-fs2 = [8e3 16e3 32e3 44.1e3 48e3 50e3];
-fsm = [0 0 0 0 1 0; ...
-       0 0 0 0 1 0; ...
```

Comment:
@iuliana-prodan any hint on how did you figure out this numbers here? Maybe small explanation in the commit message?

Reply:
@iuliana-prodan please mention the documentation link in the commit message and we are good to merge.

Reply:
My modifications are based on the documentation from https://thesofproject.github.io/latest/algos/src/sample_rate_conversion.html.

I've kept the exact vectors fs1 and fs2 as supported input and output rates and only added 96 kHz.
The fsm, the conversion matrix, I've added a new column and row with zero in the place of the non-wanted conversion and 1 for 96 convert to 48.

Reply:
@dbaluta I've update the commit message for the first 2 commits.